std/deps.ln

```diff
@@ -12,8 +12,8 @@ export event install: void
 export fn add(remote: string) {
   // TODO implement proper error handling
   const parts = remote.split('/');
-  const repo = parts[length(parts) - 1] || '';
-  const group = parts[parts.length() - 2] || '';
+  const repo = parts[length(parts) - 1 || 0] || '';
+  const group = parts[parts.length() - 2 || 0] || '';
```

Comment:
Array accesses should also be allowed to be done from Result-wrapped `int64`s, since the array access itself can fail, it could fail based on the fact that the input value has already failed, and eliminate this need to unwrap the result.

This might need some compiler changes, though, since array accesses are manually converted into certain opcode calls.